drivers/PwmOut.h

```diff
@@ -68,6 +68,7 @@ class PwmOut {
     ~PwmOut()
     {
         core_util_critical_section_enter();
+        pwmout_free(&_pwm);
```

Comment:
@ARMmbed/mbed-os-hal what was the reason not having this earlier ? I recall some discussions in some other drivers around definition of _free and implications for the rest of  targets with change like this one.

Reply:
I thought it was because not all the targets had this function. If this is the case then CI should catch any problems.

Reply:
That is it ! 

We do not have 100 percent coverage in CI, I am thinking to be on safer side to have this on 5.13 rather - to keep this on master for some time.